app/src/main/java/com/gholem/moneylab/features/planning/viewmodel/PlanningViewModel.kt

```diff
@@ -31,14 +31,9 @@ class PlanningViewModel @Inject constructor(
         Action.ShowTransactions(transactions).send()
     }
 
-    fun navigateToCategoryBottomSheet() = viewModelScope.launch {
-        navigation.emit(PlanningNavigationEvent.ToChartScreen)
-    }
-
-    fun saveNewTransactionFromPerson(positon: Int) = viewModelScope.launch {
-        val transactions = fetchTransactionModelUseCase.run(Unit)
+    fun saveNewTransactionFromCoinCap(item: TransactionModel) = viewModelScope.launch {
```

Comment:
At this point there is no relation to the `CoinCap` model (or `Exchanges after rename as I said in other comment) so the name of the method should be just `saveNewTransaction`